server/src/main/java/no/ntnu/klubbhuset/service/AdminOrganizationService.java

```diff
@@ -75,4 +76,30 @@ public boolean isAdminOfOrganization(Long organizationId) {
 
         return result;
     }
+
+    @PermitAll
+    public Response harMemberPaid(Long organizationId, User user) {
+        System.out.println("AdminOrganizationService.harMemberPaid");
+        Member member;
+        Organization organization = entityManager.find(Organization.class, organizationId);
+        TypedQuery<Member> query = entityManager.createQuery("SELECT m from Member m "
+                        + "where m.organization = :organization "
+                        + "and m.user = :user"
+                , Member.class
+        );
+        query.setParameter("user", user);
+        query.setParameter("organization", organization);
+        try {
+            member = query.getSingleResult();
+        } catch (NoResultException e) {
+            e.getMessage();
+            return Response.status(Response.Status.NO_CONTENT).entity("Is not a member").build();
+        }
+
+        if ( !member.hasPaid()) {
+            return Response.status(Response.Status.NO_CONTENT).entity("Has NOT paid").build();
+        }
```

Comment:
"Is not a member" og "Has NOT paid" bør bruke forskjellige Status-koder

Reply:
done